game.py

```diff
@@ -45,7 +45,8 @@ def move_once(t):
 
     for robot in state.get_active_robots():
         # robot.apply_card_effect(state)
-        robot.walk(3, state)
+        if not robot.inactive:
```

Comment:
tady to teď vlastně můžeš smazat když jsou to jen aktivní roboti, jak jsem to předělala ne?